src/components/help/help.stories.js

```diff
@@ -11,7 +11,7 @@ storiesOf('Help', module)
     const tooltipPosition = children ? select(
       'tooltipPosition',
       OptionsHelper.positions,
-      Help.defaultProps.tooltipPosition
+      'right'
```

Comment:
I think we need to add some styling here, rather than just changing the default option - otherwise, the overlap issue will still be present if the user switches the tooltipPosition setting manually using the knob.